src/DropDownMenu/DropDownMenu.jsx

```diff
@@ -107,7 +107,7 @@ const DropDownMenu = React.createClass({
 
   getInitialState() {
     return {
-      open: this.props.openImmediately,
+      open: false,
```

Comment:
This avoid a glitching effect with Popover when it's anchor is unset with the initial render before the `componentDidMount`'s timeout is called.